jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/ResponseHeadersTest.java

```diff
@@ -150,6 +151,44 @@ protected void doGet(HttpServletRequest req, HttpServletResponse response) throw
         assertThat("Response Header X-example", response.get("X-Example"), is(expected));
     }
 
+    @Test
+    public void testCharsetAfterGetOutputStream() throws Exception
```

Comment:
oops this was an accidental inclusion into this PR.   It is just a test to check existing behaviour.... I can remove if it irks people